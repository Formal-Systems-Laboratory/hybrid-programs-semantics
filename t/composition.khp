vars a , b ; a := a + 1 ; b := a * 2
