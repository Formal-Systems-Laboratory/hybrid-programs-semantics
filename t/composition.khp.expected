<generatedTop>
  <k>
    .
  </k>
  <state>
    a |-> ?INITIAL +Int 1
    b |-> ( ?INITIAL +Int 1 ) *Int 2
  </state>
</generatedTop>
